UI/CharactersArtworkPanel/CharactersArtworkPanel.gd

```diff
@@ -1,12 +1,17 @@
 extends Control
 
-var currentCharacter = null
-var characters = {}
+# Dude, please, set strings, arrays and other non-objects to their empty analog, not null!
+# They are uncheckable by "is_instance_valid" func.
+# Why this func?
+# Becouse checking objects by sentense "object == null" is unstable in release builds.
```

Comment:
Is it? I haven't encountered problems with it. And yeah, don't do discussions inside comments. And don't dude me ^^

Reply:
I wasn't use a github for working on project with someone (just terribly store my project backups), so I just don't know what we can discuss about code here.

Reply:
And yes, on Godot 3.3 and higer freed nodes can pass checking null sentences. [Here my issue to devs](https://github.com/godotengine/godot/issues/48642).